example/res/drawable/header_dark.xml

```diff
@@ -1,13 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android" >
-    <item>
+    <item android:top="-2dp" android:right="-2dp" android:left="-2dp">
```

Comment:
Things looked pretty ugly on the example app without changing around our backgrounds for the title items.